tests/acceptance/features/bootstrap/TagsContext.php

```diff
@@ -1342,8 +1343,12 @@ public function theHttpStatusWhenuserRequestsTagsForEntryOwnedByShouldBe(
 		$user, $fileName, $sharingUser, $status
 	) {
 		$this->requestTagsForFile($user, $fileName, $sharingUser);
+		$actualStatus = $this->featureContext->getResponse()->getStatusCode();
 		Assert::assertEquals(
-			$status, $this->featureContext->getResponse()->getStatusCode()
+			$status,
+			$actualStatus,
+			__METHOD__
+			. "Expected status is '$status' but got '$actualStatus'"
```

Comment:
```suggestion
			. " Expected status is '$status' but got '$actualStatus'"
```
There needs to be a space in the string after every use of `__METHOD__` - see PR #36954

Reply:
fixed